.travis.yml

```
@@ -116,7 +116,9 @@ jobs:
     if: repo == "ansible/ansible-lint" AND tag IS present
     python: "3.7"
     env: &deploy-env
-      TOXENV: metadata-validation
+      TOXENV: build-dists
+    before_deploy:
+      - echo > setup.py
```

This file doesn't use extra indentation for sequences.
```suggestion
    - echo > setup.py
```